lib/stripe/subscriptions/subscription_schedule.ex

```diff
@@ -0,0 +1,230 @@
+defmodule Stripe.SubscriptionSchedule do
+  @moduledoc """
+  Work with Stripe subscription schedule objects.
+
+  """
+
+  use Stripe.Entity
+  import Stripe.Request
+
+  @type plans :: %{
+          plan: String.t(),
+          quantity: pos_integer
+        }
+
+  @type phases :: %{
+          application_fee_percent: float | nil,
+          end_date: Stripe.timestamp(),
+          start_date: Stripe.timestamp(),
+          tax_percent: float | nil,
+          trial_end: Stripe.timestamp(),
+          plans: list(plans)
+        }
+
+  @type t :: %__MODULE__{
+          id: Stripe.id(),
+          object: String.t(),
+          created: Stripe.timestamp(),
+          created: String.t(),
+          canceled_at: Stripe.timestamp() | nil,
+          released_at: Stripe.timestamp() | nil,
+          completed_at: Stripe.timestamp() | nil,
+          livemode: boolean,
+          invoice_settings: %{
+            days_until_due: integer
+          },
+          current_phase: %{
+            start_date: Stripe.timestamp(),
+            end_date: Stripe.timestamp()
+          },
+          renewal_behavior: String.t(),
+          renewal_interval: String.t(),
+          revision: String.t(),
+          status: String.t(),
+          subscription: Stripe.id() | Stripe.Subscription.t(),
+          customer: Stripe.id() | Stripe.Customer.t(),
+          released_subscription: Stripe.id() | Stripe.Subscription.t() | nil,
+          phases: list(phases)
+        }
+
+  defstruct [
+    :id,
+    :object,
+    :billing,
+    :created,
+    :canceled_at,
+    :completed_at,
+    :current_phase,
+    :customer,
+    :phases,
+    :released_at,
+    :released_subscription,
+    :status,
+    :subscription,
+    :invoice_settings,
+    :livemode,
+    :renewal_behavior,
+    :renewal_interval,
+    :revision
+  ]
+
+  @plural_endpoint "subscription_schedules"
+
+  @doc """
+  Create a subscription schedule.
+
+  ### Examples
+
+  # Create subscription schedule from existing subscription
+  Stripe.SubscriptionSchedule.create(%{from_subscription: "sub_1234"})
+
+  """
+  @spec create(params, Stripe.options()) :: {:ok, t} | {:error, Stripe.Error.t()}
+        when params: %{
```

Comment:
@maartenvanvliet Last question before we merge - how did you know what params go with the `create` action?

Reply:
https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.yaml and access to the beta

Reply:
Great!  Nice work!  Just saving a mental note to look at again once out of beta.  👍